### PR TITLE
development

### PR DESCRIPTION
**Pull request by Milerius**

### CODE REVIEW

File: src/price/service.price.platform.cpp [hunk 1]
```diff
@@ -60,15 +61,38 @@ namespace antara::mmbot
...
+        all_price_.clear();
+        nlohmann::json all_price_json;
+        all_price_json["prices"] = nlohmann::json::array();
+        for (auto &&[current_pair, current_price] : res) {
+            auto current_object = nlohmann::json::object();
+            std::string current_price_str;
+            if (mmbot_config_.base_ercs_registry.at(current_pair.base.symbol.value())) {
+                current_price_str = fmt::format("{:.18f}", static_cast<double>(current_price.value()) / g_factor);
+            } else {
+                current_price_str = fmt::format("{:.8f}", static_cast<double>(current_price.value()) / g_factor);
```
Comment (DCPRevere) [inline src/price/service.price.platform.cpp]: Can we put this logic somewhere we can reuse it? As a method of the asset struct?

Comment (Milerius) [reply src/price/service.price.platform.cpp]: Yes good idea

Comment (DCPRevere) [reply src/price/service.price.platform.cpp]: I realise that it's complicated by the `base_ercs_registry`, but maybe we can put this information inside the asset struct.

Comment (Milerius) [reply src/price/service.price.platform.cpp]: the information about ercs is inside a configuration file: coins.json written by @jl777

That's why it's parsed once at the beginning of the program and can be used from everywhere

but maybe the name is bad and should be renamed

**Comment by Milerius** [comment]
@DCPRevere I Commented some part of your code that you need to update on this pull request please see: https://github.com/KomodoPlatform/antara-makerbot/pull/15/commits/b028dc96c3c934d75e40395573e1f7a34e92282d

**Review by ca333 — APPROVED** [review]

**Pull request merged**